test/functional/apps/dashboard/_view_edit.js

```diff
@@ -229,19 +206,7 @@ export default function ({ getService, getPageObjects }) {
         await PageObjects.dashboard.gotoDashboardEditMode(dashboardName);
         await PageObjects.dashboard.saveDashboard(dashboardName, { storeTimeWithDashboard: false });
         await PageObjects.dashboard.clickEdit();
-        await PageObjects.header.setAbsoluteRange('2013-10-19 06:31:44.000', '2013-12-19 06:31:44.000');
-        await PageObjects.dashboard.clickCancelOutOfEditMode();
-
-        const isOpen = await PageObjects.common.isConfirmModalOpen();
-        expect(isOpen).to.be(false);
-      });
-
-      it('when a dashboard has a filter and remains unchanged', async function () {
```

Comment:
Got rid of this test because the dashboard loaded already has a saved filter on it, it would have already broken by now.